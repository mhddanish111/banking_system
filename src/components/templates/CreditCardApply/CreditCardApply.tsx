import React from 'react';
import Layout from '../Layout';
import AccountForm from '../../organisms/AccountForm';

const CreditCardApply: React.FC<any> = () => {
  return (
    <Layout pageHeading="Apply Credit Card">
      <AccountForm type="Cards" />
    </Layout>
  );
};

export default CreditCardApply;
