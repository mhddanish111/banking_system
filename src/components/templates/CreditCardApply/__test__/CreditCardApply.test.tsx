import { shallow } from 'enzyme';
import CreditCardApply from '../CreditCardApply';

describe('<CreditCardApply />', () => {
  const wrapped = shallow(<CreditCardApply />);
  it('should render the CreditCardApply Component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });
});
