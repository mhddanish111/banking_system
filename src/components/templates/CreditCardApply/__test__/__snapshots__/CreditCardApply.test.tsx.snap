// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreditCardApply /> should render the CreditCardApply Component correctly 1`] = `
<Layout
  pageHeading="Apply Credit Card"
>
  <AccountOpenForm
    type="Cards"
  />
</Layout>
`;
